pkg/socketenricher/tracer_test.go

```diff
@@ -89,6 +103,15 @@ func TestSocketEnricherBind(t *testing.T) {
 		}
 		return
 	}
+	stringToSlice4096 := func(s string) (ret [4096]uint8) {
+		for i := 0; i < 4096; i++ {
+			if i >= len(s) {
+				break
+			}
+			ret[i] = uint8(s[i])
+		}
+		return
+	}
```

Comment:
That does not work: https://goplay.tools/snippet/zZ1NrbL9qBT

But copy() can work when used with `[:]`: https://goplay.tools/snippet/4ak3ppcJx8k

Reply:
Actually, I can't make it work with copy() because a string is like a `[]byte` but the array is not an array of `byte`:
https://goplay.tools/snippet/Z412DV-6P1B